src/bindgen/ir/documentation.rs

```diff
@@ -57,6 +57,14 @@ impl Source for Documentation {
             return;
         }
 
+        if config.language == Language::Zig {
+            for line in &self.doc_comment[..end] {
+                write!(out, "//{}", line);
```

Comment:
Doc comments in Zig are triple slashes `///`, is there any reason for using a standard comment here?